### DISCUSSION TITLE
Monitoring Service Health

### DISCUSSION THREAD

**Discussion by surajit-zs** [gofr-dev/gofr#430] [Q&A] [answered]
`/.well-known/health-check`

not able to use this endpoint getting errors like this 
```bash
{"error":{"message":"http: no such file"}}
```

**Comment by aryanmehrotra** [comment, marked as answer]
You can use `/.well-known/health` - this has to be fixed in documentation as well.

It is being tracked here - https://github.com/gofr-dev/gofr/issues/433
